HTML+CSS+JS/Background/Auto-Scrolling-Background/style.css

```diff
@@ -0,0 +1,43 @@
kIi5yNV97457DcZljEq17yWU46oJRC724Z4XpY/qnUo2uMFIbs1g126/Fc0J7egDSsen0RlsEqLwxWaVKHz/6r6f+q+3vUwLqg7E7WWRhG3JNgFpjaYw0sm/VC+4/Bsm0y9zS49FGeisxgrX4jrEUJVk3iAU5yWDGDFYkNrMIiNA3LZj6LjFWGgf8ELMs5TbeHwbnoxrdsGMxgjf+gZQ1rYOUj1akrLLikvXPAfm8P6Y+3GR86/B5Y1nkVL0lg4R/Yaa2PpQBr3mG87wjL7xCFFLA6WxaJSImKCN5MI2ztt8PCs2RWD2CQ08qy2gzWvgTrdzr4OsvKZzt22BVWcp+tc0PysaBXYUE2a2fZSScd0Bu6Yc9orTS0wqqEbLc2B89CxCLeboPVOpG2++spr/PLsNLzguUic5D4KsAqOfge3r0clO7KM+ljXejAYGUPjc4p0215r8MiETzgI/gRm36+CisoTizA2pRSnGj1Mqxyarlu7RGMWR9xsgsBM6Qarb9gWcJEGohzwxdgaWCc9UNjKMAalWG9Pjes5C9qglIyZGa09K3LGgchcJdrq2iAvGUJD+TLsOAxshksLslDfJnQQCfSX9yFA7R+uaK2q5tI51GNwSclgPdFWI0pmq/C0jhYYalo6MjXwbLmxiXDqi0dAm3PRhrMs/xusLLvGDceFnH8Zbe8eu0tBuBRHirOdbl8OKP3QZ+bceBs+Muw+Ev8Pp8VrQRYQTkbjM8vLc4Gy8/yiQZ1HRrk997HXFbiW8EqRsNICB00WA60Xi1YjCrnmdcWLMAHGw+tRZhXd74XrDxDySrkDFalkfizrZDQIHIDHatE7jWL2O2+vohZYQD+BVjWhAUrT2BBL88zTvjpDukWTnkDCjEJ6SIrmTaVzhI59VNy0l+LWb2zv5iQruDV4o5xVr8f5+YoCYtEdkkc4Jof3IMR5oYgPeCRuSzr7IuwKiUsOuZLw5pUdt7hppIafGTjMnXzhn0/3Oa3w/YzB1gPK2QwjYOvZaXrbgULj53ZWiffDOK9zV2rgKUPJnQpAdB2bL6kZ263KN+vKssUsH2Ri0zdeF0l3lRCJR6BTxZYumE4epFxq4cF/EV+wHn82Bw3nWEtCyd5Hm82x63u8NcqYNk6vu8f18ctvxlnLyT/aLKgussXW82F9ZrWu6N+dfdrfUE6uWLct/jiQ9/maxENsDRW9bdty4kwmgagI6zgxuCQb2Ic6eK1OFjUL2DMFZKMsAQLgFO1pbg36/xqCxjM9Joz/GqO1uBo3bSarQkW6bxCckK/d4WVRXaN12rJwetbMQdP5BbRJJ39Z+4ZDeOn75mh9Vzo+jWb9R2neUJOrgxGfUeobHFyGmD5AuAKrJaKtGlVHg3LSpRz8OJR+aopbpkkS9pF9mF+OLOWI7QPtfY3ZRFSWlj7ugwHr9syrsD1B07t87TwOYR1sDQwQlybJSxLc0eUlgCCXNwuwbLX3HpE+kvLyWsI/ALcLFuKxoEW+I88OYzswSzZLVADjK6mAqY/G9j1m6HQvBlV0iBzdjeM/May/wx9sfKF9uoFLPBYc2XpFBablZQsi+XMjGxhyGatc/ZEhuD9JgFvjGjbEljWejdOcLGDLGtsFojy08M43SyHaYwYj718XRrxo737cRYuq/OWwLtcx3fctM0uanbv7PrQHA1t8n26xyJZXUA863Dkpa+1gRfHMai4AASAyyA5nq7bQnRQd/c4OcggIy//ICYo/bLTz0Z+EE4xdpzkOhitx6Hr0eVfhrOjMcg2+efCg+HYQsmeD3rUaVdZ65D+MzMtrG80CLyQK2JEyQ6nVRkWOKLCv9Ua1vNED3DdIJyt5sPt4D443MhDCVyXGTGg25+ohKye64+O8+FhuH9MUoZm9Sh6IPuY3QTpBv5sTC40OOw3l8S5mEt6ACguFdDlRsFlQ86/Hc4fxAVxV+bvgq01xmhvupNx8PFsuwQwy2vnS7C65XkADQGTew0CTdxoVF32nrY6qfUnoSN/UOn+aj6m36TX0fJVmfnvudVHACaH5ReogVUUxBxiEsQyL5awSazSpeiahTZY6NY5TJPebvjyn1fvciEgUOQ+5rEvc1uiQz6JkDkUuHZtkHtv7oQvrUr9ziptzhyxpkfIh/4u4JYwo2strPLySt6wXkryfGOVtv0GebUB7Yn38CdjlE6NiYOvrSRWFhhxrA7v9jKt8oZy8xeL3XEMSaRC5uZrOuKdbqPaplfqz0UntF/Ih31vVV5VMGGTYXQkI4GveZs4nOxiL6xf+llaP8+xsl4u035bVQYUyJw6Xsck3IFxb+Jet0uzYX8nCCqp0Yx142ry/68qsIDGnDo+m8S2aOkzHgWNmw4aYKGxJhEH/E9U8+KegM1z0CB5OyRIQrOGltck3Xt0At22KfN/q7pXQpkDRus005Jd5CCZMNd+vxYWGrzha3tqYSUvsct6IlofJ/7SDJZeOKm1lVpYb/mKI60eFq0uMgIYRaeP6f3jbH3WFh7qRkPC6g37oNYAS4MkdGcMaDqaqiusCI3f066aYJGRbNcrLZxc1JbzK7AQHr3fMJip+aWu4UAo9zSsXy7DQvfL+71kjKnldcHmkU+rd4KF0P5d31FK1fYiapcYV06iYbE3Dwuj++SN36zcCovmx2cLVo9/CitCp+vbTZ1FPdkL63oPJ3vP2K9WWBg5x3d+YXeip38dxTU3U1qTQIv60CGYIlrNmD7Md/87A89h0Vy0ORmeDbvh9SFgY/fOh8nybeMFTl1y/mQe7V3iphhD8y8e+Bl/26ljgaRpHp396ieQ0jrDUqJSsCSkYElIGtZPZqssS0IKloQULAkpWBJSsCSkYElIwZKQgiUhBUtCCpaEFCwJKVgSUrAkpGBJSMGSkIIlIQVLQgqWhBQsCSlYElKwJKRgSUjBkpCCJSEFS0IKloQULAkpWBJSsCSkYElIwZKQgiUhBUtCoPy3gZRa9B+ovCyGraEZmgAAAABJRU5ErkJggg==")
```

Comment:
@Pranav016 What do you think regarding this? Shall we go for a local image from component folder itself or this seems fine?

Reply:
Will incorporate this change asap
I am thinking of using this image now
![image](https://user-images.githubusercontent.com/65714751/125675731-eaec8bd4-8c14-4982-90f1-a2a9e96247a0.png)

Should I go with this ?

Reply:
ok. Try with it.